tests/congruence_tests/test_query.py

```diff
@@ -414,14 +422,13 @@ def dense_many_discover(cls, client: QdrantBase) -> List[models.ScoredPoint]:
     def dense_context_image(cls, client: QdrantBase) -> List[models.ScoredPoint]:
         return client.query_points(
             collection_name=COLLECTION_NAME,
-            query=models.ContextQuery(
-                context=models.ContextPair(positive=11, negative=19)
-            ),
+            query=models.ContextQuery(context=models.ContextPair(positive=11, negative=19)),
             with_payload=True,
-            limit=10,
+            limit=1000,
```

Comment:
is there a smaller value that works? 
1000 seems overkill

Reply:
we can decrease the collection size, we need to check all the points, because there might be score overlap which will lead to a failure